package.json

```diff
@@ -52,7 +52,7 @@
     "rename-package": "node -r ./scripts/ts-node-register ./scripts/rename-package.ts",
     "run:published": "node ./scripts/monorepo/runPublished.js",
     "runto:lerna": "node ./scripts/monorepo/runTo.js",
-    "satisfied": "satisfied --skip-invalid --ignore \"angular|sass|svelte|@testing-library|vue\"",
+    "satisfied": "satisfied --skip-invalid --ignore \"angular|lit|sass|svelte|@testing-library|vue|web-components\"",
```

Comment:
What's the error about web-components?

Reply:
The error is [here](https://github.com/microsoft/fluentui/pull/19166#issuecomment-888593642) and this is with regards to `@storybook/web-components` and not our own 😆

Reply:
That's weird. Can you add specifically @storybook/web-components then? And does everything seem to work okay without it?

Reply:
Will do! And yeah all stories for v8, converged packages and web-components load just fine without it; there's also nothing in the storybook migration notes/changelogs that suggests `@storybook/web-components` is mandatory to run stories.